### PR TITLE
Issue 14 - Add option to select content type to use glossary

### LINKED ISSUE #14: Add option to select which content types should show tooltips
Currently tooltips are shown on all content types and that makes no sense as displaying them, for instance, on covers is not a good idea.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #14

### CODE REVIEW

File: src/collective/glossary/browser/views.py [hunk 1]
```diff
@@ -98,9 +98,14 @@ def __init__(self, context, request):
 
     def tooltip_is_enabled(self):
         """Check if term tooltip is enabled."""
-        return api.portal.get_registry_record(
+        context = self.context
+        enable_tooltip = api.portal.get_registry_record(
             IGlossarySettings.__identifier__ + '.enable_tooltip'
         )
+        enabled_content_types = api.portal.get_registry_record(
```
Comment (hvelarde) [inline src/collective/glossary/browser/views.py]: @rodfersou this condition should not be here; this is redundant with the `is_glossary_object()`

File: src/collective/glossary/upgrades/v2/__init__.py [hunk 2]
```diff
@@ -9,3 +9,10 @@ def update_resource_conditions(setup_tool):
     setup_tool.runImportStepFromProfile(profile, 'cssregistry')
     setup_tool.runImportStepFromProfile(profile, 'jsregistry')
     logger.info('Conditions to include package resources were updated.')
+
+
+def update_controlpanel_options(setup_tool):
+    """Update control panel options."""
+    profile = 'profile-{0}:default'.format(PROJECTNAME)
+    setup_tool.runImportStepFromProfile(profile, 'registry')
```
Comment (hvelarde) [inline src/collective/glossary/upgrades/v2/__init__.py]: @rodfersou you should use `setup_tool.runImportStepFromProfile(profile, 'plone.app.registry')`

File: src/collective/glossary/interfaces.py [hunk 3]
```diff
@@ -21,6 +22,16 @@ class IGlossarySettings(form.Schema):
         default=True,
     )
 
+    enabled_content_types = schema.List(
+        title=_(u'Enabled Content Types'),
+        description=_(u'Only objects of these content types will apply the glossary'),
+        required=False,
+        default=DEFAULT_ENABLED_CONTENT_TYPES,
```
Comment (hvelarde) [inline src/collective/glossary/interfaces.py]: @rodfersou by default, all content types but Glossary and Term, should be enabled

Comment (rodfersou) [inline src/collective/glossary/interfaces.py]: as defined at IRC we will enable by default all default Plone content types

**Pull request merged**